news-bundle/src/Controller/Page/NewsFeedController.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\NewsBundle\Controller\Page;
+
+use Contao\CoreBundle\Asset\ContaoContext;
+use Contao\CoreBundle\Controller\AbstractController;
+use Contao\CoreBundle\Routing\Page\DynamicRouteInterface;
+use Contao\CoreBundle\Routing\Page\PageRoute;
+use Contao\CoreBundle\ServiceAnnotation\Page;
+use Contao\NewsBundle\Event\FetchArticlesForFeedEvent;
+use Contao\NewsBundle\Event\TransformArticleForFeedEvent;
+use Contao\PageModel;
+use FeedIo\Feed;
+use FeedIo\Specification;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @Page(contentComposition=false)
+ */
```

Comment:
```suggestion
#[AsPage(contentComposition: false)]
```

You also have to add a `use` statement for the class.

Reply:
Fixed in d8ddd2965